mayachain/chain.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mayachain",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mayaprotocol.com/",
+  "pretty_name": "Maya Protocol",
+  "chain_id": "mayachain-mainnet-v1",
+  "bech32_prefix": "maya",
+  "daemon_name": "mayanode",
+  "node_home": "$HOME/.mayanode",
+  "slip44": 931,
+  "codebase": {
+    "git_repo": "https://gitlab.com/mayachain/mayanode",
+    "recommended_version": "V1.103.3",
+    "compatible_versions": [
+      " mainnet "
+    ],
+    "genesis": {
+      "genesis_url": " https://tendermint.mayachain.info/genesis "
+    },
+    "versions": [
+      {
+        "name": " mainnet ",
+        "recommended_version": " mainnet ",
```

Comment:
this info (recommended and compatible) should be the same as above in this case.